### DISCUSSION TITLE
Preloading fragment content

### DISCUSSION THREAD

**Discussion by Zimbelstern** [hakimel/reveal.js#3414] [Q&A]
Hi,

my presentations contain fragments with images. These bitmap data seem to be fetched or rendered only at the moment the fragment is activated, which leads to the distortion of the fade transition and does not look very appealing. When you go back and activate the fragment again, the transition works as desired.

Is there any way to preload these image data?

**Comment by t-fritsch** [comment]
HI @Zimbelstern,

I never experienced such thing with fragment images, could you please provide an example of your markup ? If possible a working codesandbox/jsfiddle/codepen that shows the problem would be great 👍
